style.css

```diff
@@ -0,0 +1,12 @@
+
+.bord {
+    background-color: rgba(197, 194, 194, 0.4);
+
+    
```

Comment:
I think we are supposed not to leave multiple empty lines according to the best practice of Html and Css.
Again, Linters will detect this as an error

Reply:
- Nice